text/0000-recycle-vec.md

```diff
@@ -205,6 +205,13 @@ directly subtyping relations between two generic types in where clauses. [1]
 Thus, it was deemed to be sufficient to limit verification of the memory layout to the checking
 of size and alignment of the stored type.
 
+There is an alternative to this API, provided by https://crates.io/crates/vec_utils,
+that instead of panicking on mismatching size or alignment, just allocates a new `Vec`
```

Comment:
`vec-utils` does not allocate a new `Vec`, it just returns `Vec::new()` if the layouts are incompatible

Reply:
Ah, indeed, `Vec::new()` doesn't allocate.